rmake.py

```diff
@@ -210,8 +216,20 @@ def config_cmd():
     create_dir( os.path.join(build_path, "clients") )
     os.chdir( build_path )
 
+    # compiler
+    if args.use_hipcc_compiler:
+        cmake_options.append(f"-DCMAKE_C_COMPILER={rocm_path}/bin/hipcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER={rocm_path}/bin/hipcc")
+    else:
+        cmake_options.append(f"-DCMAKE_C_COMPILER=gcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER=g++")
+
     if args.static_lib:
         cmake_options.append( f"-DBUILD_SHARED_LIBS=OFF" )
+        # force hipcc for static libs
+        cmake_options.append(f"-DCMAKE_C_COMPILER={rocm_path}/bin/hipcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER={rocm_path}/bin/hipcc")
+        print("Forcing compiler to hipcc for static library.")
```

Comment:
Can you document the reason for this?

Reply:
Seems to build fine with g++ for me now, so not sure why I put this originally. I'll do some searching through emails/jira and see if I come up with anything.

Reply:
I removed this since it seems to work.